.github/workflows/tests.yml

```diff
@@ -0,0 +1,37 @@
+# This workflow will install Python dependencies, run tests and lint with a variety of Python versions
+# For more information see: https://help.github.com/actions/language-and-framework-guides/using-python-with-github-actions
+
+name: Run tests
+
+on:
+  push:
+    branches: [ master ]
+  pull_request:
+    branches: [ master ]
```

Comment:
This will only run tests on PRs which target `master`, meaning that if you have a chain of PRs only the root one will get test runs. That miiight be what you want, but probably isn't.

If you're happy for tests to run on all PRs you can just drop this line:
```suggestion
```

Reply:
Thanks!